src/webgl/p5.Geometry.js

```diff
@@ -185,17 +185,142 @@ p5.Geometry = class Geometry {
     return n.mult(Math.asin(sinAlpha) / ln);
   }
   /**
- * computes smooth normals per vertex as an average of each
- * face.
- * @method computeNormals
- * @chainable
- */
-  computeNormals() {
+   * This function calculates normals for each face, where each vertex's normal is the average of the normals of all faces it's connected to.
+   * i.e computes smooth normals per vertex as an average of each face.<br>
+   * When using 'FLAT' shading, vertices are disconnected/duplicated i.e each face has its own copy of vertices.<br>
```

Comment:
Right now for other methods like `createFramebuffer` (https://p5js.org/reference/#/p5/createFramebuffer), rather than passing in the string `'FLAT'`, we create a variable in constants.js like `const FLAT = 'flat'` and then in the code, check if the parameter is equal to `constants.FLAT`. That way users can pass it in as just `FLAT` (without quotes). Do you think we can do that for flat/smooth too for consistency?